local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -94,6 +98,16 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 	return nil
 }
 
+// GetFsAvailable returns available capacity about a mounted filesystem. fullPath
+// is the pathname of any file within the mounted filesystem.
+func (u *volumeUtil) GetFsAvailable(fullPath string) (uint64, error) {
+	var s syscall.Statfs_t
+	if err := syscall.Statfs(fullPath, &s); err != nil {
+		return 0, err
+	}
+	return uint64(s.Frsize) * s.Bavail, nil
```

Comment:
What's the difference between frsize and bsize?

Do we want to use total blocks instead of available?  I'm thinking of a situation where they might want pre-populated data in the volume. 

Also in the shared directory case, we could get inconsistent capacities depending on the order that pods fill them up (although the capacity is already going to be inaccurate in that case).